src/Base/Renderer.hpp

```diff
@@ -17,5 +18,5 @@ class Renderer {
          * @param backend The backend to use for rendering.
          * @return The created renderer.
          */
-        static Renderer* makeRenderer(BACKEND backend);
+        static Renderer* makeRenderer(BACKEND backend, Window & createdWindow);
```

Comment:
Undocumented parameter.
Misleading variable name (the window hasn't been created yet).